### PR TITLE
Update RPM docs warning about support

### PR DESCRIPTION
**Pull request by sheajamba**
## Description

Updates the wording of the warning for RPM usage.

See: https://github.com/trinodb/trino/pull/14524

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/installation/rpm.rst [hunk 1]
```diff
@@ -13,7 +13,8 @@ to start as a service.
 
   It is recommended to deploy Trino with the Helm chart on Kubernetes or
   manually with the :doc:`Docker containers <containers>` or the :doc:`tar
-  archive <deployment>`. While the RPM is available for use, it is not tested.
+  archive <deployment>`. While the RPM is available for use, it is not
+  supported.
```
Comment (hashhar) [inline docs/src/main/sphinx/installation/rpm.rst:17]: avoid "supported". It has specific meaning to different people. I'd probably suggest saying something like "using the RPM is discouraged in favour of the tarball or Docker containers".

**Review by findepi — APPROVED** [review]

File: docs/src/main/sphinx/installation/rpm.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/installation/rpm.rst:17]: I like that.

**Comment by mosabua** [comment]
Also @sheajamba .. seems like there is a conflict.. might have to rebase

**Review by mosabua — APPROVED** [review]
Thanks .. looks good now.

**Comment by mosabua** [comment]
@findepi or @hashhar .. can we get this merged please?

**Comment by findepi** [comment]
i am reluctant to merge since the CI is not green

**Comment by mosabua** [comment]
I just checked this out and built the docs locally. It all works. The CI failure is wrong @findepi

**Comment by findepi** [comment]
If CI is wrong, it may require to be fixed

**Comment by mosabua** [comment]
> If CI is wrong, it may require to be fixed

Sure .. but this is in io.trino:trino-main and has nothing to do with the docs .. maybe we can just kick off a build again or so? 

@colebow is currently looking a disabling flaky tests and such .. maybe this is one of them.

**Comment by colebow** [comment]
@findepi - I've been looking into CI failures and this is the exact failure that's causing most of the problems on Trino. It looks to me like an issue with Maven downloading a dependency for a plugin, but there is no test failure - it's something wrong with our infra. I haven't been able to repro locally.

**Comment by findepi** [comment]
> Sure .. but this is in io.trino:trino-main and has nothing to do with the docs ..

If this was true, why would we run it as part of the docs build then?


> maybe we can just kick off a build again or so?

i think we already tried. it looked deterministic to me, but feel free to add another empty commit here to rerun it.
if it comes green, awesome

**Comment by mosabua** [comment]
@sheajamba can you push again so CI runs again .. maybe it works this time.

**Comment by sheajamba** [comment]
Pushed again and now all have passed.

**Comment by mosabua** [comment]
@findepi  .. as we suggested .. the failure was a false alarm. Please merge now.

**Pull request merged**